src/main/java/com/manardenza/console/VisualUserMenu.java

```diff
@@ -0,0 +1,67 @@
+package com.manardenza.console;
+
+import com.manardenza.Main;
+import com.manardenza.entity.AbstractObject;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+
+public class VisualUserMenu {
+
+    private org.slf4j.Logger log = LoggerFactory.getLogger(VisualUserMenu.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the "log" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1450)